server/src/QRServer/common/classes.py

```diff
@@ -186,6 +186,7 @@ class RankingEntry:
     user_id: str
     wins: int
     games: int
+    rating: int = 0
```

Comment:
I really don't like this, it messes up other places and is used in only one - on the other hand it is needed in that one place. Perhaps a separate structure, like `ExtendedRankingEntry` would be ok to use here instead? It'd basically be a copy of this one + this rating field

Reply:
To remove